### DISCUSSION TITLE
HxGrid with dynamic Columns

### DISCUSSION THREAD

**Discussion by stayboy** [havit/Havit.Blazor#311] [General]
How possible is HxGrid for supporting dynamic Columns using RenderFragment blazor component to render dynamic columns

**Comment by stayboy** [comment]
Resolved. It works

**Comment by hakenr** [comment]
Just for record, you can
* use `Visible` property of the `HxGridColumn`,
* use `@if` and other logic in `Column` definition to render columns,
* generate the `Columns` RenderFragement on your own,
* ...

**Reply by stayboy** [reply]
So awesome... Great job guys
